Add explanation of reporting vulnerabilities (#1471)
fluent/fluentd-kubernetes-daemonset: SECURITY.md

# Security Policy

## Supported Versions

| Version   | Supported          |
|-----------|--------------------|
| 1.16.x    | :white_check_mark: |
| <= 1.15.x | :x:                |

## Reporting a Vulnerability

> If you found the vulnerability about default configuration of **fluentd-daemonset-SOMETHING**, report from the following page:

Reviewer: How about this?

```suggestion
If you find a vulnerability of **fluentd-daemonset-SOMETHING** with the default configuration, report it from the following page:
```

Author: Thanks!

* https://github.com/fluent/fluentd/security/advisories

> [!IMPORTANT]

> > As fluentd-kubernetes-daemonset images are downstream of [fluent/fluentd](https://hub.docker.com/r/fluent/fluentd) container.

Reviewer: I believe this 'As' is not necessary.

```suggestion
> fluentd-kubernetes-daemonset images are downstream of [fluent/fluentd](https://hub.docker.com/r/fluent/fluentd) container.
```

> > Thus, even though security scanner reports pile of vulnerabilities, the updated container image can't be shipped

Reviewer: ```suggestion
> Thus, even though security scanner reports a pile of vulnerabilities, the updated container image can't be shipped
```

> > until updated container image is deployed from upstream at first.

Reviewer: `at first` doesn't sound right in this context.

```suggestion
> until updated container image is deployed from upstream first.
```

> * If you found that bundled Ruby gems which are related to **fluentd-daemonset-SOMETHING** have vulnerabilities, please report to [fluentd-kubernetes-daemonset](https://github.com/fluent/fluentd-kubernetes-daemonset/issues/new).
> 
> * If you found that bundled Ruby gems which are related to **fluentd-docker-image** have vulnerabilities, please report to [fluentd-docker-image](https://github.com/fluent/fluentd-docker-image/issues/new).

Reviewer: `which are` seems to be misleading. 
(`that`に比べると単なる付加情報を足すものらしいです、ぼくもよくわかりませんが、、省略できそうなので省略しちゃったほうがよさそうだなと思いました)

```suggestion
* If you find that bundled Ruby gems related to **fluentd-daemonset-SOMETHING** have vulnerabilities, please report to [fluentd-kubernetes-daemonset](https://github.com/fluent/fluentd-kubernetes-daemonset/issues/new).

* If you find that bundled Ruby gems related to **fluentd-docker-image** have vulnerabilities, please report to [fluentd-docker-image](https://github.com/fluent/fluentd-docker-image/issues/new).
```
